docs/frameworks/symfony.md

```diff
@@ -127,8 +143,28 @@ Since Symfony 4, the production parameters are configured through environment va
 provider:
     environment:
          APP_ENV: prod
+         TRUSTED_PROXIES: '127.0.0.1'
+         TRUSTED_HOSTS: '^api\.example\.com$'
```

Comment:
Where are these environment variables used? I can't see them in the diff.

Reply:
They are used in the Symfony default index.php

https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/5.1/public/index.php

Reply:
Oh I think I understand from the diff above:

> That could be done with [environment variables](#Environment variables) or by running 
these lines in `public/index.php`.

Simply setting those variables is enough? I would say that it's better to document one solution, it is simpler and less confusing. The env var seem quite simple to setup (compared to editing Symfony code), what do you think?

Reply:
Thanks for the link, turns out we commented at the same time.

What do you think about recommending only the environment variables? (for simplicity)